Postman/Postman.js

```diff
@@ -1,54 +1,49 @@
-"use strict";
+const Q = require('q');
+const _ = require('underscore');
+const moment = require('moment-timezone');
```

Comment:
__[jshint]__
*  'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).